tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3510,96 +3502,14 @@ public function deleteGroup(string $group):void {
 	 * @throws GuzzleException
 	 */
 	public function groupHasBeenDeleted(string $group):void {
-		if (OcisHelper::isTestingWithGraphApi()) {
-			$this->graphContext->adminHasDeletedGroupUsingTheGraphApi($group);
+		if ($this->groupExists($group) && $this->isTestingWithLdap()) {
+			$this->deleteLdapGroup($group);
 		} else {
-			$this->deleteGroup($group);
+			$this->graphContext->adminHasDeletedGroupUsingTheGraphApi($group);
 		}
 		$this->groupShouldNotExist($group);
 	}
 
-	/**
-	 * @When /^the administrator deletes group "([^"]*)" from the default user backend$/
-	 *
-	 * @param string $group
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function adminDeletesGroup(string $group):void {
-		$this->deleteGroup($group);
-		$this->pushToLastStatusCodesArrays();
-	}
-
-	/**
-	 * @When /^the administrator deletes group "([^"]*)" using the provisioning API$/
-	 *
-	 * @param string $group
-	 *
-	 * @return void
-	 * @throws Exception
-	 */
-	public function deleteTheGroupUsingTheProvisioningApi(string $group):void {
-		$this->emptyLastHTTPStatusCodesArray();
-		$this->emptyLastOCSStatusCodesArray();
-		$this->response = UserHelper::deleteGroup(
```

Comment:
Also, remove the UserHelper::deleteGroup function definition, which is not used anymore.

Reply:
Okay